Mathlib/Tactic.lean

```diff
@@ -77,6 +77,7 @@ import Mathlib.Tactic.GeneralizeProofs
 import Mathlib.Tactic.Group
 import Mathlib.Tactic.GuardGoalNums
 import Mathlib.Tactic.GuardHypNums
+import Mathlib.Tactic.HashCommandLinter
```

Comment:
I wonder if we should import this in `Mathlib.Tactic.Common`, since it uses no theory files, to get it earlier in the hierarchy? What's the reasoning behind importing it here, and/or do you know if this has been discussed anywhere?

I'm also wondering if the file itself should be in new folders: Mathlib > Command > Linter > HashCommand.lean. It's a bit odd for me to see a linter in `Tactic`, but `GuardMsgs` was in `Std.Tactic`, so there is precedent for command-related things being in `Tactic`...though I think it might be nice to start having linters in their own folder one way or another! I'd like to start the discussion on zulip unless it's been discussed already.

But! I won't hold up this PR on this basis. We can always move things around later. :)

Reply:
I already have an accompanying PR where I import the linter in `Mathlib.Tactic.Basic`, since I wanted to test the linter on (most of) Mathlib.

As for moving linters to a separate folder, I think that this is a good idea, but maybe I would leave that for a second PR: there is also a `Tactic.Lint` file that should probably move there as well.

Reply:
The PR with this linter imported early should be #11028.

Reply:
Sounds good! :)